app/routers/audio.py

```diff
@@ -0,0 +1,376 @@
+async def get_choices(
+    session: Session = Depends(get_db),
+    new_user: User = Depends(get_placeholder_user),
+    music_on: bool = Form(...),
+    music_choices: Optional[List[str]] = Form(None),
+    music_vol: Optional[int] = Form(None),
+    sfx_on: bool = Form(...),
+    sfx_choice: Optional[str] = Form(None),
+    sfx_vol: Optional[int] = Form(None)
+        ) -> RedirectResponse:
+    """This function saves users' choices in the db.
+
+    Args:
+        request (Request): the http request
+        session (Session): the database.
+        new_user (User): default user.
+        need to be replaced with real one,
+        after registration is implemented.
+        music_on_off (str, optional): On if the user chose to enable music,
+        false otherwise.
+        music_choices (Optional[List[str]], optional): a list of music tracks
+        if music is enabled, None otherwise.
+        music_vol (Optional[int], optional): a number in the range (0, 1)
+        indicating the desired music volume, or None if disabled.
+        sfx_on_off (str, optional): On if the user chose to enable
+        sound effects, false otherwise.
+        sfx_choice (Optional[str], optional): chosen sound effect for
+        mouse click if sound effects are enabled, None otherwise.
+        sfx_vol (Optional[int], optional): a number in the range (0, 1)
+        indicating the desired sfx volume, or None if disabled.
+
+    Returns:
+        RedirectResponse: redirect the user to home.html.
+    """
+    user_choices = (
+        {"music_on": music_on, "music_vol": music_vol,
+         "sfxs_on": sfx_on, "sfxs_vol": sfx_vol})
+    save_audio_settings(
+        session, new_user, music_choices, sfx_choice, user_choices)
+
+    return RedirectResponse("/", status_code=HTTP_302_FOUND)
+
+
+@router.get("/start")
+async def start_audio(session: Session = Depends(get_db),) -> RedirectResponse:
+    """Starts audio according to audio settings.
+
+    Args:
+        session (Session): the database.
+
+    Returns:
+        RedirectResponse: redirect the user to home.html.
+    """
+    (music_on, playlist, music_vol,
+        sfxs_on, sfx_choice, sfxs_vol) = get_audio_settings(session)
+    if music_on is not None:
+        music_vol = handle_vol(
+            music_on, music_vol)
+        sfxs_vol = handle_vol(
+            sfxs_on, sfxs_vol)
+
+    if not playlist:
+        playlist = DEFAULT_MUSIC
+        music_vol = DEFAULT_MUSIC_VOL
+
+    if not sfx_choice:
+        CHOSEN_SFX = DEFAULT_SFX
+        CHOSEN_SFX_VOL = DEFAULT_SFX_VOL
+    else:
+        CHOSEN_SFX = sfx_choice
+        CHOSEN_SFX_VOL = sfxs_vol
+
+    return json.dumps(
+        {"music_on": music_on,
+            "playlist": playlist,
+            "music_vol": music_vol,
+            "sfxs_on": sfxs_on,
+            "sfx_choice": CHOSEN_SFX,
+            "sfxs_vol": CHOSEN_SFX_VOL})
+
+
+# Functions for audio setup according to users' choices:
+
+
+def init_audio_tracks(
```

Comment:
I think it's better to create an object called `Sound`, which have field of type `SoundKind`, which as an enum of `SFX` and `SONG`. The object should be serializeable.

The current design is a good start, but can be clanky in case of more complex usages